safe/gui/widgets/test/field_mapping_tab_example.py

```diff
@@ -0,0 +1,42 @@
+# coding=utf-8
+"""Example usage of field mapping widget."""
+
+import sys
+from safe.test.utilities import get_qgis_app, load_test_vector_layer
+
+QGIS_APP, CANVAS, IFACE, PARENT = get_qgis_app()
+
+from PyQt4.QtGui import QApplication, QWidget, QGridLayout
+
+from safe.gui.widgets.field_mapping_tab import FieldMappingTab
+from safe.definitions.field_groups import age_ratio_group
```

Comment:
__[flake8]__
* [E402] module level import not at top of file

<!-- Comment by [SideCI](https://sideci.com) -->